content/source/docs/cloud/users-teams-organizations/single-sign-on/okta.html.md

```diff
@@ -48,3 +50,29 @@ Be sure to copy the metadata URL (from the final step of configuring Okta) befor
 6. Your Okta SSO configuration is complete and ready to [use](../single-sign-on.html#using-sso).
 
     ![sso-settings](../images/sso/settings-okta.png)
+
+## Team Mapping Configuration (Okta)
+
+Terraform Cloud can automatically add users to teams based on their SAML assertion, so you can manage team membership in your directory service. To do this, you must specify the `MemberOf` SAML attribute, and make sure the AttributeStatement in the SAML Response contains a list of AttributeValue items in the correct format (i.e., comma-separated list of team names). For additional details on this and other SSO concepts within the context of Terraform Cloud, please refer to [this](../single-sign-on.html) overview page.
+
+If you haven't yet completed all steps outlined in the [Configuration (Okta)](#configuration-okta-) section above, please do so before proceeding.
+
+To enable this automated team mapping functionality, edit your Terraform Cloud Okta Application and complete the following steps:
+
+1. Expand the "Attributes" section of the Application configuration (under the "Sign On" tab):
+
+    ![The button for expanding the customizable Attributes section in the Okta portal appears to the left of the text "Attributes (optional)"](../images/sso/okta-attribute-settings.png)
+
+2. Set the `Group Attribute` Statements to the following:
```

Comment:
```suggestion
2. Set the "Group Attribute Statements" to the following:
```

Same as previous tweak. The values below can stay the same though, since they're verbatim text or selections that need to be entered  by the user. 👍🏼